src/commandSocket.cpp

```diff
@@ -629,7 +629,18 @@ void CommandSocket::sendPrintTimeMaterialEstimates()
     logDebug("Sending print time and material estimates.\n");
     auto message = std::make_shared<cura::proto::PrintTimeMaterialEstimates>();
 
-    message->set_time(FffProcessor::getInstance()->getTotalPrintTime());
+    std::unordered_map<PrintFeatureType, double> time_estimates = FffProcessor::getInstance()->getTotalPrintTimes();
```

Comment:
Okay because its an enum class you have to cast it to unsigned char first though.